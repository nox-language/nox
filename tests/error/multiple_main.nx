fun main() = print "hello"

fun main() = print "hello 2"
