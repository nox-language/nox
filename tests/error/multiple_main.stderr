error: Duplicate function `main`
   --> tests/error/multiple_main.nx:3:1
  |
3 | fun main() = print "hello 2"
  | ^^^^^^^^

